### PR TITLE
[15.0] [MIG] account move update analytic

### PR DESCRIPTION
**Pull request by Shide**
Sttraight migration of https://github.com/OCA/account-analytic/pull/497 + fix https://github.com/OCA/account-analytic/pull/507 directly (I've spotted this while migrating and I'm the same user, so don't need to slow the process)

@rafaelbn  @remi-filament please review

MT-1778 @moduon

### CODE REVIEW

**Review by rafaelbn — COMMENTED** [review]
- [x] We need to track fields account analytic and account analytic tags in the same way Odoo track accounts
imagen

**Review by rafaelbn — COMMENTED** [review]
- [ ] We need to change multiple lines at once for fields account analytic and account analytic tags in the same way Odoo allow to change accounts from:
     - Invoicing - Accounting -> Journals
          - Sales
          - Purchase
          - Bank and cash
          - Miscellaneous

imagen

**Review by remi-filament — APPROVED** [review]
LGTM !

**Comment by Shide** [comment]
@rafaelbn Since this is a straight migraiton with an small fix, I'll keep the tracking fields suggestion but I've discarded the multi edit lines.

File: account_move_update_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Authors
+~~~~~~~
+
+* Le Filament
+
+Contributors
+~~~~~~~~~~~~
+
+* Rémi - Le Filament <remi-filament>
```
Comment (rafaelbn) [inline account_move_update_analytic/README.rst:63]: Please add us, we have dedicated a bunch of hours and in fact we are going un improve it more with functions and test

- Eduardo de Miguel ([Moduon](https://www.moduon.team/))
- Rafael Blasco ([Moduon](https://www.moduon.team/))

File: account_move_update_analytic/README.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+1. In a posted move, go to the move line you want to update, and click on 'Update analytic' to open the wizard. (You also have the button directly on account move lines)
```
Comment (rafaelbn) [inline account_move_update_analytic/README.rst:38]: 1. In a posted move and from customer invoices and vendor bills, go to the move line you want to update, and click on 'Update analytic' to open the wizard. (You also have the button directly on account move lines).

File: account_move_update_analytic/README.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+2. Current analytic is shown, you can select another one then click on Confirm
```
Comment (rafaelbn) [inline account_move_update_analytic/README.rst:39]: 2. Current analytic account and tags are shown, you can select another one then click on Confirm

**Review by loida-vm — APPROVED** [review]
Tested and approved. LGTM!

File: account_move_update_analytic/README.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+3. Analytic lines will be dropped from current analytic account and recreated on new analytic account/tags.
+
```
Comment (rafaelbn) [inline account_move_update_analytic/README.rst:41]: Please add this images in this PR o the next one. Thank you! (You can use the github URL)

imagen

imagen

**Review by rafaelbn — COMMENTED** [review]
Minor comments for readme, tested functionaly 👍 in runboat

File: account_move_update_analytic/README.rst (hunk 2 above)
Comment (Shide) [reply account_move_update_analytic/README.rst:38]: README file will be updated with the contents of readme folder, so don't need to add them in README explicitly.

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Pull request merged**